plugins/outputs/warp10/README.md

```diff
@@ -0,0 +1,19 @@
+# README #
+
+Telegraph plugin to push metrics on Warp10
+
+### Telegraph output for Warp10 ###
+
+Execute a post http on Warp10 at every flush time configured in telegraph in order to push the metrics collected
+
+### Config ###
+
+Add following instruction in the config file (Output part)
+
+```
+[[outputs.warp10]]
+warpUrl = "http://localhost:4242/api/v0/update"
```

Comment:
I think we should have the user only specify the base URL `http://localhost:4242`, then we can tack on `/api/v0/update`.  This will be a bit easier for the user and matches other plugins.  However, I wouldn't do this if its common for the path to differ, for example due to proxy use.

What about TLS configuration?  We generally have this for all HTTP plugins and we try to support it in all new plugins, does Warp10 support this?